code/game/objects/items/weapons/tanks/tank_types.dm

```diff
@@ -251,3 +214,41 @@
 	..()
 	air_contents.oxygen -= (3*ONE_ATMOSPHERE)*volume/(R_IDEAL_GAS_EQUATION*T20C)
 	air_contents.toxins = (10*ONE_ATMOSPHERE)*volume/(R_IDEAL_GAS_EQUATION*T20C)
+
+/*
+ * Air Mix
+ */
+/obj/item/tank/air
+	name = "air tank"
+	desc = "Mixed anyone?"
+	icon_state = "air"
+	item_state = "air"
+	distribute_pressure = ONE_ATMOSPHERE*O2STANDARD
```

Comment:
It appears the default distribute pressure is in fact, 101 kPa, changing it to 21 kPA will result in choking people to death

Reply:
distribute_pressure = ONE_ATMOSPHERE*O2STANDARD
So that's...101.325 *0.21 =21.27825

So, if we use Air Mix, that's 50/50 Oxygen/Nitrogen. That's 10.639125kpa Oxygen and 10.639125kpa Nitrogen. Great. Everyone's dead. Thanks, maths.

Thank you, working on it.